packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -944,7 +917,7 @@ export default class InternalModel {
       if (CUSTOM_MODEL_CLASS) {
         this.store._notificationManager.notify(this.identifier, 'relationships');
       } else {
-        this._record.notifyHasManyAdded(key);
+        this._record.notifyPropertyChange(key);
```

Comment:
it's a similar to change to the other PRs, same effect, once rebased the change will only have occurred once. Basically just ensures we shunt the notification through notifyPropertyChange instead of the misdirection.